src/cljs/replumb/repl.cljs

```diff
@@ -47,15 +47,22 @@
   "Given a namespace return all the public var analysis maps. Analagous to
   clojure.core/ns-publics but returns var analysis maps not vars."
   ([ns]
-   (ns-publics env/*compiler* ns))
-  ([state ns]
    {:pre [(symbol? ns)]}
    (->> (merge
-         (get-in @state [:cljs.analyzer/namespaces ns :macros])
-         (get-in @state [:cljs.analyzer/namespaces ns :defs]))
+         (get-in @st [:cljs.analyzer/namespaces ns :macros])
+         (get-in @st [:cljs.analyzer/namespaces ns :defs]))
         (remove (fn [[k v]] (:private v)))
         (into {}))))
 
+(defn ns-interns
```

Comment:
Yes, it does not work but we need to override Clojure's `ns-interns` functionality.